packages/vscode-extension/src/utils/commonUtils.ts

```diff
@@ -115,8 +115,8 @@ export function getProjectId(): string | undefined {
   }
 }
 
-export async function isSPFxProject(workspacePath: string): Promise<boolean> {
-  if (await fs.pathExists(`${workspacePath}/SPFx`)) {
+export function isSPFxProject(workspacePath: string): boolean {
+  if (fs.pathExistsSync(`${workspacePath}/SPFx`)) {
```

Comment:
Why is async version replaced with sync version?

Reply:
Because sendTelemetryEvent API doesn't have a async version, so I can't use await in this function.

Reply:
Ok, I think this value should be kept in memory instead of reading from disk every time when we send telemetry event.

Reply:
I've considered about this solution, but I'm afraid the value might change. For example, user first init an app, then add spfx and at this time, it'll change to spfx project.